packages/commerce-sdk-react/src/hooks/utils.ts

```diff
@@ -42,22 +45,24 @@ export const updateCache = <Action extends ShopperCustomersMutationType>(
                 : item === cacheQuery.queryKey[index]
         )
 
-    // STEP 1. Update data inside query cache for the matching queryKeys
-    queryKeysMatrix[action](params, response)?.update?.map((queryKey: any) => {
-        queryClient.setQueryData(queryKey, response)
-    })
+    if (queryKeysMatrix && queryKeysMatrix[action]) {
```

Comment:
Minor nitpick: could be 

`if (queryKeysMatrix?.[action]) {`

Reply:
Ah I forgot to remove this - by using `queryKeysMatrix[action]?.` we actually don't need this if statement anymore